rcsb/utils/validation/ValidationReportReader.py

```diff
@@ -85,7 +85,7 @@ def __buildCif(self, rD, containerName="vrpt"):
         curContainer = DataContainer(containerName)
         for elName in rD:
             catName = elName
-            if (not rD[elName]) or (not self.__attribD[catName]) or (catName in ["programs"]):
+            if (not rD[elName]) or (catName not in self.__attribD) or (not self.__attribD[catName]) or (catName in ["programs"]):
```

Comment:
Maybe using `.get()` would be simpler?
```suggestion
            if (not rD[elName]) or (not self.__attribD.get(catName)) or (catName in ["programs"]):
```